packages/web-components/fast-foundation/src/toolbar/toolbar.template.ts

```diff
@@ -23,19 +23,24 @@ export const toolbarTemplate: (
         aria-orientation="${x => x.orientation}"
         orientation="${x => x.orientation}"
         role="toolbar"
-        @click="${(x, c) => x.clickHandler(c.event as MouseEvent)}"
-        @focusin="${(x, c) => x.focusinHandler(c.event as FocusEvent)}"
-        @keydown="${(x, c) => x.keydownHandler(c.event as KeyboardEvent)}"
     >
         <slot name="label"></slot>
         <div class="positioning-region" part="positioning-region">
             ${startTemplate}
-            <slot
-                ${slotted({
-                    filter: elements(),
-                    property: "slottedItems",
-                })}
-            ></slot>
+            <span
+                class="content"
+                part="content"
+                @focusin="${(x, c) => x.focusinHandler(c.event as FocusEvent)}"
+                @click="${(x, c) => x.clickHandler(c.event as MouseEvent)}"
+                @keydown="${(x, c) => x.keydownHandler(c.event as KeyboardEvent)}"
+            >
```

Comment:
I think there is an assumption that start/end content *isn't* meant to be part of the toolbar - is that assumption accurate? I'm not sure - considering the role of toolbar is on the host, I would assume that start/end are "regions" here I can still drop slotted content.